python/dgl/distributed/dist_graph.py

```diff
@@ -157,14 +157,16 @@ def __getitem__(self, key):
         return NodeSpace(data=NodeDataView(self._graph, key))
 
 class HeteroEdgeView(object):
-    """A NodeView class to act as G.nodes for a DistGraph."""
+    """A EdgeView class to act as G.edges for a DistGraph."""
     __slots__ = ['_graph']
 
     def __init__(self, graph):
         self._graph = graph
 
     def __getitem__(self, key):
-        assert isinstance(key, str)
+        assert isinstance(key, str) or (
+            isinstance(key, tuple) and len(key) == 3
```

Comment:
How about 
```suggestion
            isinstance(key, (tuple, list)) and len(key) == 3
```

Reply:
canonical etype is always in tuple, should not be a list.

Reply:
At first I thought that there could be cases there they might be created by the user, but that makes sense